examples/transcribe_from_microphone.py

```diff
@@ -0,0 +1,67 @@
+import speechmatics
+import speechmatics.models
+import speechmatics.client
+import speechmatics.cli
+import asyncio
+import argparse
+import sys
+import sounddevice as sd
+
+
+class RawInputStreamWrapper:
+    def __init__(self, wrapped: sd.RawInputStream):
+        self.wrapped: sd.RawInputStream = wrapped
+
+    def read(self, frames):
+        return bytes(self.wrapped.read(frames)[0])
+
+
+async def transcribe_from_device(device, speechmatics_client, language: str):
+    frame_rate = 44_100
+    with sd.RawInputStream(
+        device=device, channels=1, samplerate=frame_rate, dtype="float32"
+    ) as stream:
+        settings = speechmatics.models.AudioSettings(
+            sample_rate=frame_rate,
+            chunk_size=1024 * 4,
```

Comment:
where does this chunk size value come from? maybe add  a comment?

Reply:
I removed it now, it was set to the default value anyway